nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -315,17 +345,21 @@ void reportInitializerError(
       Symbol.MethodSymbol methodSymbol,
       String message,
       VisitorState state,
-      Description.Builder descriptionBuilder) {
+      Description.Builder descriptionBuilder,
+      List<Symbol> fixable) {
```

Comment:
Here instead of `fixable` how about `nonNullFields`?  And again, update Javadoc

Reply:
Changed it, thank you. f2c044fb80078bff8f8015e59c7815a2f8149609